instrumentation-api-annotation-support/src/main/java/io/opentelemetry/instrumentation/api/annotation/support/MethodSpanAttributesExtractorBuilder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.annotation.support;
+
+/** A builder of {@link MethodSpanAttributesExtractor}. */
+public final class MethodSpanAttributesExtractorBuilder<REQUEST, RESPONSE> {
+  MethodExtractor<REQUEST> methodExtractor;
+  MethodArgumentsExtractor<REQUEST> methodArgumentsExtractor;
+  ParameterAttributeNamesExtractor parameterAttributeNamesExtractor;
```

Comment:
Hmm, since all 3 params are required, WDYT about using a static factory method instead of a builder?

Reply:
Good idea.  I was trying to make it fit into the same builder pattern as `Instrumenter` but I agree that it's not a good fit.